unit-test

```diff
@@ -14,9 +14,10 @@ build=$(mktemp -d -p $here)
 cd $build
 
 ../cmake-init.sh
-ninja aktualizr-lite t_lite-helpers libt_lite-mock.so uptane-generator make_ostree_sysroot
+ninja aklite-tests -j6
```

Comment:
ninja can't automatically determine this?

Reply:
apparently yes, just get used to `make` and never realized that ninja can do it for you :)